pkgs/applications/editors/emacs/elisp-packages/manual-packages/emacs-application-framework/default.nix

```diff
@@ -46,7 +47,8 @@ let
       git
       nodejs
       wmctrl
-      xdotool
+      (lib.optionals (lib.meta.availableOn stdenv.hostPlatform xdotool)
```

Comment:
One semantic error: `lib.optionals (..) [ xdotool ]` or `lib.optional (..) xdotool`.  Note the `s` in the function name.

One CI error: please format this file with `nixfmt`.

---

This is technically right.  But putting it one level higher would be easier to read.

```
  otherPackageLists = [
    (
      [
        git
        nodejs
        wmctrl
      ]
      ++ lib.optional (lib.meta.availableOn stdenv.hostPlatform xdotool) xdotool
    )
  ]
  ++ appOtherDeps;
```